symja_android_library/pom.xml

```diff
@@ -106,8 +106,8 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<java.version>11</java.version>
 		<release.build>false</release.build>
-		<deployment.suppress>false</deployment.suppress> <!-- child modules can
-		set this true to not be published -->
+		<deployment.suppress>false</deployment.suppress> <!-- child modules can set this true to not be published -->
+		<guava.minimal.version>32.0</guava.minimal.version>
```

Comment:
@axkr can you tell what the minimally required Guava version for Symja is?
I don't think it is important to find the very latest version, since at least for me it is more important to also allow future versions of Guava without a release of Symja.